lib/jekyll/admin.rb

```diff
@@ -1,7 +1,26 @@
 require "jekyll/admin/version"
-
+require "jekyll/admin/admin_servlet"
+require "jekyll/admin/api_servlet"
+require "jekyll/admin/api_handler"
+require "jekyll/admin/configuration/handler"
+require "jekyll/admin/configuration/servlet"
+require "jekyll/admin/pages/handler"
+require "jekyll/admin/pages/servlet"
+require_relative "commands/serve"
 module Jekyll
   module Admin
-    # Your code goes here...
+    # Path to the compiled frontend folder
+    def self.public_path
+      File.join(File.dirname(Dir.pwd), 'jekyll-admin/lib/jekyll/admin/public/dist').to_s
+    end
+
+    # Returns a hash of route and the respective handler
+    def self.api_servlet_bindings
```

Comment:
Are these always going to get `Jekyll.sites.first`? If so, maybe it'd make more sense moving that into `serve.rb` to DRY things up a bit?